src/main/resources/io/jenkins/plugins/extendedsecuritysettings/ExtendedSecuritySettings/help-httpHeaderFilters.html

```diff
@@ -0,0 +1,40 @@
+<!--
+  ~ The MIT License
+  ~
+  ~ Copyright (c) 2019 CloudBees, Inc.
+  ~
+  ~ Permission is hereby granted, free of charge, to any person obtaining a copy
+  ~ of this software and associated documentation files (the "Software"), to deal
+  ~ in the Software without restriction, including without limitation the rights
+  ~ to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+  ~ copies of the Software, and to permit persons to whom the Software is
+  ~ furnished to do so, subject to the following conditions:
+  ~
+  ~ The above copyright notice and this permission notice shall be included in
+  ~ all copies or substantial portions of the Software.
+  ~
+  ~ THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+  ~ IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+  ~ FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+  ~ AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+  ~ LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+  ~ OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+  ~ THE SOFTWARE.
+  -->
+
+<div>
+    Configure a set of HTTP header names to filter out from unauthorized responses.
+    Header names are <em>case-insensitive.</em>
+    Users lacking the <i>Overall/Read</i> permission will have these headers removed from HTTP responses.
+    Example headers that can be filtered include:
+    <ul>
+        <li><kbd>X-Jenkins</kbd></li>
+        <li><kbd>X-Jenkins-Session</kbd></li>
+        <li><kbd>X-Jenkins-CLI-Port</kbd></li>
+        <li><kbd>X-Jenkins-CLI2-Port</kbd></li>
+        <li><kbd>X-Instance-Identity</kbd></li>
+        <li><kbd>X-SSH-Endpoint</kbd></li>
+        <li><kbd>X-Hudson</kbd></li>
+        <li><kbd>X-Hudson-CLI-Port</kbd></li>
+    </ul>
```

Comment:
I would be inclined to filter this list back down to a smaller set. I don't know that it helps the readability to have a long list like this. Unless we want to tell the user that this is a good set of things to filter, which goes beyond what I think we want to say.

Reply:
Those are basically all the weird headers I saw on ci.jenkins.io